src/translations/ro.json

```diff
@@ -0,0 +1,21 @@
+{
+  "error-occured": "O eroare a împiedicat extensia din a functiona corect: ",
+  "version-update-message": "Folosești ezpp! v{0}",

```

Comment:
just to make sure, this is the message displayed when the extension is updated (this has caused some confusion previously). does this message work when displayed as, for example, "Folosești ezpp! v1.6.0"?

Reply:
Yes. That's how it should work.